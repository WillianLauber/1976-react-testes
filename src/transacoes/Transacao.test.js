import React from "react";
import { render } from "@testing-library/react";
import Transacao from "./Transacao";

describe("Componente de transacao do extrato", () => {
  it("O snapshot do componente sempre deve permanecer o mesmo", () => {
    const { container } = render(
      <Transacao data="17/01/2022" tipo="saque" valor="28.00" />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
