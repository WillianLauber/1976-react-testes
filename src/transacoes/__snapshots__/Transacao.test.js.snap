// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transacao do extrato O snapshot do componente sempre deve permanecer o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    17/01/2022
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    28.00
  </p>
</div>
`;
